padertorch/contrib/examples/audio_synthesis/wavenet/data.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+from padercontrib.database.librispeech import LibriSpeech
+from padertorch.contrib.je.data.transforms import AudioReader, STFT, \
+    fragment_signal, Collate
+
+
+def get_datasets(audio_reader, stft, max_length, batch_size, shuffle_test=False):
+    db = LibriSpeech()
```

Comment:
can you change this so you are not using the LibriSpeech Database object but the lazy.database.JsonDatabase with a json_path which is preset to the NT_DATABASE_JSON_PATH / librispeech.json?